components/channel_invite_modal/channel_invite_modal.jsx

```diff
@@ -36,20 +39,42 @@ export default class ChannelInviteModal extends React.Component {
     constructor(props) {
         super(props);
 
+        this.onChange = this.onChange.bind(this);
+        this.onStatusChange = this.onStatusChange.bind(this);
+        this.onHide = this.onHide.bind(this);
+        this.handleInviteError = this.handleInviteError.bind(this);
+        this.handlePageChange = this.handlePageChange.bind(this);
+        this.search = this.search.bind(this);
+        this.addValue = this.addValue.bind(this);
+        this.handleDelete = this.handleDelete.bind(this);
+        this.renderOption = this.renderOption.bind(this);
+        this.renderValue = this.renderValue.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
Can you remove all these binds and use arrow function syntax for all the class functions?